networkit/cpp/centrality/TopCloseness.cpp

```diff
@@ -345,10 +346,14 @@ void TopCloseness::run() {
     });
     tlx::d_ary_addressable_int_heap<node, 2, Aux::LessInVector<double>> Q{farness};
 
-    std::vector<node> nodes;
-    nodes.reserve(G.numberOfNodes());
-    G.forNodes([&](const node u) { nodes.emplace_back(u); });
-    Q.build_heap(std::move(nodes));
+    if (nodeList.empty()) {
+        std::vector<node> nodes;
+        nodes.reserve(G.numberOfNodes());
+        G.forNodes([&](const node u) { nodes.emplace_back(u); });
+        Q.build_heap(std::move(nodes));
```

Comment:
You can probably use
```suggestion
        Q.build_heap(G.nodeRange().begin(), G.nodeRange().end());
```

Reply:
Good point.